src/Kunstmaan/TranslatorBundle/Service/Command/Importer/Importer.php

```diff
@@ -42,6 +48,67 @@ public function import(\Symfony\Component\Finder\SplFileInfo $file, $force = fal
         return $importedTranslations;
     }
 
+    /**
+     * @param string $file
+     * @param array $locales
+     * @param bool $force
+     *
+     * @return int
+     */
+    public function importFromSpreadsheet(string $file, array $locales, $force = false)
+    {
+        $filePath = realpath(dirname($file)) . DIRECTORY_SEPARATOR;
+        $fileName = basename($file);
+        $file = $filePath . $fileName;
+
+        if (!file_exists($file)) {
+            throw new LogicException(sprintf('Can not find file in %s', $file));
+        }
+
+        $format = (new File($file))->guessExtension();
+        if (!Type::XLSX == $format && !Type::CSV == $format && !Type::ODS == $format) {
```

Comment:
This check will not work like this.

Reply:
also, !==, it looks confusing having a == but with a ! at the start